libff/algebra/fields/fp12_2over3over2.tcc

```diff
@@ -275,6 +275,59 @@ Fp12_2over3over2_model<n, modulus> Fp12_2over3over2_model<n, modulus>::
         my_Fp6(z0, z4, z3), my_Fp6(z2, z1, z5));
 }
 
+// bls12-381 from scipr-lab (VV)
+template<mp_size_t n, const bigint<n> &modulus>
+Fp12_2over3over2_model<n, modulus> Fp12_2over3over2_model<n, modulus>::
+    mul_by_045(
+        const Fp2_model<n, modulus> &ell_0,
+        const Fp2_model<n, modulus> &ell_VW,
+        const Fp2_model<n, modulus> &ell_VV) const
+{
+    /*
+    // OLD
+    Fp12_2over3over2_model<n,modulus> a(my_Fp6(ell_VW, my_Fp2::zero(),
+    my_Fp2::zero()), my_Fp6(my_Fp2::zero(), ell_0, ell_VV));
+
+    return (*this) * a;
+    */
+
+    // (VV)
```

Comment:
Ok there seem to have plenty of these.
Please make sure to remove all comments like `bls12-381 from scipr-lab (VV)` (at least before this branch can be considered for merging) and remove the `VV` initials. If a comment is useful we keep it and formulate it clearly for the broader audience to understand (no need to have initials).

Reply:
Sorry, the VV tag was added just for me for debug. I thought I've removed it everywhere before commit. Apparently not. Now hopefully yes.